environment-api/src/main/java/com/sequenceiq/environment/api/v1/environment/endpoint/EnvironmentEndpoint.java

```diff
@@ -50,17 +50,11 @@
     @ApiOperation(value = EnvironmentOpDescription.GET, produces = MediaType.APPLICATION_JSON, notes = ENVIRONMENT_NOTES, nickname = "getEnvironmentV1")
     DetailedEnvironmentResponse get(@PathParam("name") String environmentName);
 
-    @DELETE
-    @Path("/{name}")
-    @Produces(MediaType.APPLICATION_JSON)
-    @ApiOperation(value = EnvironmentOpDescription.DELETE, produces = MediaType.APPLICATION_JSON, notes = ENVIRONMENT_NOTES, nickname = "deleteEnvironmentV1")
-    SimpleEnvironmentResponse delete(@PathParam("name") String environmentName);
```

Comment:
Please don't remove the single delete. Maybe the CLI and the UI don't use it right now, other clients might need this basic functionality. We should provide the clients an easy-to-use API.

Reply:
Also we should create a proper RESTful API and that needs this method.